### PR TITLE
docs, app/vmgateway: add description about new `auth.httpHeader` flag

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Feature was implemented in the enterprise code.

Related issue: https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3054

### CODE REVIEW

**Comment by dmitryk-dk** [comment]
Replaced to enterprise

**Review by tenmozes — APPROVED** [review]
LGTM

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ The following tip changes can be tested by building VictoriaMetrics components f
...
 * FEATURE: [MetricsQL](https://docs.victoriametrics.com/MetricsQL.html): add [sort_by_label_numeric](https://docs.victoriametrics.com/MetricsQL.html#sort_by_label_numeric) and [sort_by_label_numeric_desc](https://docs.victoriametrics.com/MetricsQL.html#sort_by_label_numeric_desc) functions for [numeric sort](https://www.gnu.org/software/coreutils/manual/html_node/Version-sort-is-not-the-same-as-numeric-sort.html) of input time series by the specified labels. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2938).
 * FEATURE: [vmbackup](https://docs.victoriametrics.com/vmbackup.html) and [vmrestore](https://docs.victoriametrics.com/vmrestore.html): retry GCS operations for up to 3 minutes on temporary failures. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3147).
+* FEATIRE: [vmgateway](https://docs.victoriametrics.com/vmgateway.html): define HTTP header name to look for JWT authorization token. It helps in such a case when user can't pass a jwt token at the same time because it should be passed in the same http header. See [feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3054).
```
Comment (valyala) [inline docs/CHANGELOG.md:36]:
```suggestion
* FEATURE: [vmgateway](https://docs.victoriametrics.com/vmgateway.html): add ability to extract JWT authorization token from non-standard HTTP header by passing it via `-auth.httpHeader` command-line flag. See [this feature request](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/3054).
```

**Review by valyala — APPROVED** [review]

**Comment by valyala** [comment]
@dmitryk-dk , thanks for the pull request!

**Pull request merged**